transactions/update-contract/2023/dec-6-upgrade-dkg-staking-epoch/update_contract_set_access_slots.cdc

```diff
@@ -0,0 +1,10 @@
+transaction(name: String, code: String) {
+	prepare(signer: AuthAccount) {
+
+		let slots: {UInt8: UInt16} = {5:5}
```

Comment:
@vishalchangrani Do we know how many slots we want to keep open for access nodes each week?

Reply:
Lets start with 3 instead of 5. This can be updated in the future, correct?

Reply:
yes, it can be updated